src/patterns/components/message/warning.hbs

```diff
@@ -0,0 +1,12 @@
+---
+order: 3
+name: Warning Message
+sourceless: true
+---
+
+<div class="Message Message--warning">
+  You seem to be offline at the moment. <span class="u-hidden u-md-inlineBlock">We&rsquo;ll keep trying to connect.</span> <a href=".">Try again</a>
+  <button class="Message-close">
+    {{svg "icons/x" class="Icon Icon--large"}}
+  </button>
+</div>
```

Comment:
If we edit our base message template per my earlier comment, this could be rewritten as:

```hbs
{{#embed "patterns.components.message.overview" 
  messageClass="Message--warning"
  includeClose=true}}
  {{#content "content"}}
    <p>You seem to be offline at the moment. <span class="u-hidden u-md-inlineBlock">We&rsquo;ll keep trying to connect.</span> <a href=".">Try again</a></p>
  {{/content}}
{{/embed}}
```

Now if we need to modify the markup structure, the close button semantics, or anything like that, it'll trickle down to every instance.